### PR TITLE
[HSEARCH] 7.1.0.CR1

### PR DESCRIPTION
**Pull request by marko-bekhta**
https://staging.hibernate.org/search/releases/7.1/

### CODE REVIEW

File: search/releases/7.1/index.adoc [hunk 1]
```diff
@@ -106,4 +106,106 @@ indexDescriptor.field( "releaseDate" ).ifPresent( field -> {
...
+SearchScope<Person> personSubTypesScope = searchMapping.scope( Person.class,
+        List.of( "Manager", "Associate" ) ); // <3>
+----
+<1> link:{hsearch-doc-url-prefix}#entrypoints-search-mapping[Retrieve the `SearchMapping`].
+<2> Create a `SearchScope` targeting only the `Book` entity type using a class.
+<3> Create a `SearchScope` targeting only the `Manager` and `Associate` subtypes using their entity names.
+====
+
+See this section of the reference documentation on the link:{hsearch-doc-url-prefix}#entrypoints-search-scope[ways to create search scope]
+for more information.
```
Comment (yrodiere) [inline search/releases/7.1/index.adoc]: You're pushing it, it's not that important of a feature xD

File: _data/projects/search/releases/7.1/series.yml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 summary: >-
   Vector search in both Lucene and Elasticsearch backends,
   allow looking up the capabilities of each field in the metamodel,
+  new query string predicate,
+  simplify entity registration in the Standalone POJO mapper,
+  allow targeting entities by name in the Standalone POJO Mapper
```
Comment (yrodiere) [inline _data/projects/search/releases/7.1/series.yml]: Same here, not sure it's worth mentioning everywhere we talk about the series :)

**Review by yrodiere — APPROVED** [review]
LGTM, a few comments but that's up to you.

File: search/releases/7.1/index.adoc (hunk 1 above)
Comment (marko-bekhta) [reply search/releases/7.1/index.adoc]: well ooooooooooooook 😃

File: _data/projects/search/releases/7.1/series.yml (hunk 2 above)
Comment (marko-bekhta) [reply _data/projects/search/releases/7.1/series.yml]: I'll keep it in CR1 since I've kept it for the blog as well 🙈

**Pull request merged**